src/System.Net.Mail/tests/Functional/SmtpServer.cs

```diff
@@ -0,0 +1,126 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+//
+// SmtpServer.cs - Dummy SMTP server used to test SmtpClient
+//
+// Author:
+//   Raja R Harinath <harinath@hurrynot.org>
+//
```

Comment:
@AlexGhiondea, is this considered a copyright that needs to be kept?
https://github.com/mono/mono/blob/0bcbe39b148bb498742fc68416f8293ccd350fb6/mcs/class/System/Test/System.Net.Mail/SmtpServer.cs#L1-L6